core/src/main/kotlin/org/evomaster/core/search/service/Archive.kt

```diff
@@ -390,6 +390,11 @@ class Archive<T> where T : Individual {
      */
     fun addIfNeeded(ei: EvaluatedIndividual<T>): Boolean {
 
+        // TODO: Disabled, since it is not necessary
```

Comment:
actually, this should be put back.
if there is a deth sentence, do not add individual.
this logic could still be used in other contexts.
however, we do not give anymore death sentence if a test starts a WM instance